services-speech/src/main/java/com/mapbox/api/speech/v1/MapboxSpeech.java

```diff
@@ -0,0 +1,178 @@
+package com.mapbox.api.speech.v1;
+
+
+import com.google.auto.value.AutoValue;
+import com.mapbox.core.exceptions.ServicesException;
+import com.sun.istack.internal.Nullable;
+
+import java.io.File;
+
+import okhttp3.Cache;
+import okhttp3.OkHttpClient;
+import okhttp3.ResponseBody;
+import retrofit2.Call;
+import retrofit2.Callback;
+import retrofit2.Response;
+import retrofit2.Retrofit;
+
+import static com.mapbox.core.constants.Constants.BASE_API_URL;
+
+@AutoValue
+public abstract class MapboxSpeech {
+  /**
+   * Makes the API call to get the instruction using parameters specified when MapboxSpeech was
+   * built.
+   *
+   * @param instruction text to pass to API voice for dictation
+   * @param callback to alert status updates for API call
```

Comment:
All public APIs in Mapbox Java use the `@since` annotation inside the javadoc. Many methods are missing this.

Reply:
@cammace add 3.0.0?

Reply:
yeah so the annotations `@since 3.0.0`